src/compat/db_query.c

```diff
@@ -1089,7 +1089,7 @@ db_dump_query_result (DB_QUERY_RESULT * r)
     {
       fprintf (stdout, "Query_id: %lld \n", (long long) r->res.s.query_id);
       fprintf (stdout, "Stmt_id: %d \n", r->res.s.stmt_id);
-      fprintf (stdout, "Tuple Cnt: %d \n", r->res.s.cursor_id.list_id.tuple_cnt);
+      fprintf (stdout, "Tuple Cnt: %lld \n", r->res.s.cursor_id.list_id.tuple_cnt);
```

Comment:
Is it not necessary to consider the difference between %lld and %I64d?

fprintf (stdout, "Tuple Cnt: %" PRId64 " \n", r->res.s.cursor_id.list_id.tuple_cnt);

Reply:
I think other codes in CUBRID prefer to print INT64 with "%lld". I'm going to keep it.

Reply:
I found they cast to (long long int) for INT64, and (unsigned long long int) for UINT64 when printing with `%lld` and `%llu` respectively. I fixed them. Thank you.